app/controllers/signup.js

```diff
@@ -335,14 +342,20 @@ export default class SignupController extends Controller {
     registerUser(user)
       .then((res) => {
         if (res.status === 204) {
+          identifyUser();
+          trackEvent('User Registered - New SignUp Flow');
           window.open('https://realdevsquad.com/goto', '_self');
         } else {
+          trackEvent('Unable to Sign Up - New SignUp Flow Breaks');
           res.json().then((res) => {
             this.errorMessage = res.errors[0].title;
```

Comment:
What does [0] refers to? Is it possible to have a errors object instead of array? 

If not you can separate it like
```suggestion
            const <errorName> = res.errors[0];
            this.errorMessage = <errorName>.title;
```

Reply:
What is advantage of this change?

Reply:
We can extract more properties if we need to (in future), with this separation

Reply:
Made the change